description.md

```diff
@@ -18,8 +17,34 @@ Each topic in this file contains a small description and links to the files cove
 
 ### Assignments
 
-### Drills
+Assignments are practical activities that learners undertake individually or in teams over the span of multiple weeks.
+The primary objective of an assignment is to create a compelling scenario that allows learners to apply the concepts they have learned.
+
+#### Support
+
+Assignments are typically meant for individual work, but they often require support from trainers.
+This support aims to clarify requirements and prevent students from getting stuck on tasks.
+
+Going beyond basic assistance, forums can be implemented as public platforms where learners can openly share questions, provide answers, and engage in discussions without revealing explicit solutions.
+
+#### Difficulty
+
+Each assignment should employ a gradual increase in difficulty, fostering self-evaluation and the development of critical thinking skills.
```

Comment:
By self-evaluation I guess you actually mean that the assignment comes with a checker script, not that the students evaluate themselves, but the meaning is the latter.